assets/component-slider.css

```diff
@@ -32,6 +32,24 @@ slider-component {
   }
 }
 
+@media screen and (min-width: 750px) {
+  .slider.slider--tablet-up {
+    position: relative;
+    flex-wrap: inherit;
+    overflow-x: auto;
+    scroll-snap-type: x mandatory;
+    scroll-behavior: smooth;
+    scroll-padding-left: 1rem;
+    -webkit-overflow-scrolling: touch;
+    margin-bottom: 1rem;
+  }
+
+  .slider.slider--tablet-up .slider__slide {
+    margin-bottom: 0;
+    padding-bottom: 0;
+  }
```

Comment:
I think you might need to edit the negative margin applied to the `.grid` class here by using a padding-left value or resetting the margin on the `product__media-list` class ? 
On smaller window size (I was testing 915px) the media image is bigger than the container which makes the thumbnail list look not aligned: 
![](https://screenshot.click/21-12-2od6f-rqf53.png)